spec/models/post_spec.rb

```diff
@@ -0,0 +1,60 @@
+# Tests Comments
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'Validations' do
+    let(:user) do
+      User.create(name: 'Rishi', photo: 'Image will be displayed here', bio: 'Hello guys', posts_counter: 0)
+    end
+    let(:post) do
+      Post.create(title: 'First Post', text: 'This is my first post', author_id: user.id, comments_counter: 0,
+                  likes_counter: 0)
+    end
+
+    it 'Must have a title' do
+      post.title = nil
+      expect(post).to_not be_valid
+    end
+
+    it 'Invalid if more than 250 characters' do
+      post = Post.create(
+        title: 'sdghjashgdjahsgjdahsgjdahgsjdsgjashdgjshgjash
+        gdjashgdjashgddfdjhkjdhkjdhfkjdhksjdhfyujashgdjashgd
+        jashgdjashgvcbvndasdasdasdadvcvbnmeurioeuroeirueoirueoriue
+        ordjashdgasjhdgjashdgjashdgagellodfdhjdhkfjoeimamsdhst
```

Comment:
- [ ] Dear @Rishi-Mishra0704, You have done a great job adding the tests, but please remember we need to test all our custom methods in our model, as part of this, Please add a test for your `recent_five_comments` method inside your `Post` model.